cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SpatialTest.scala

```diff
@@ -257,9 +258,63 @@ class SpatialTest extends DocumentingTest {
           |<<cypher-spatial-crs, four coordinate reference systems>>.
           |This allows for both <<schema-index-use-a-single-property-index-with-where-using-equality, equality>>
           |and <<schema-index-use-index-with-where-using-range-comparisons, range>> queries using exactly the same syntax and behaviour as for other property types.
+          |If two range predicates are used, setting minimum and maximum points, this will effectively result in a
```

Comment:
Reword - this is clunky. Maybe "If two range predicates are used **by** setting minimum and maximum points," or similar

Reply:
I think you got that back to front. The predicates result in min,max, not the other way round.